### PR TITLE
Leeloo v2 | Addition of rev2 firmware files, and clean-up of rev1 files.

### PR DESCRIPTION
**Pull request by ClicketySplit**
## Description
Created the required rev2 files to support Leeloo v2, and tried to consolidate some of the configuration line items in the config.h files into info.json files according to the hierarchy.

Updated the default/keymap.c to include some helpful features; and, added default_led/keymap.c to include some features to manage RGB Underglow.

## Types of Changes
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR
N/A

## Checklist
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/clickety_split/leeloo/keymaps/default/rules.mk [hunk 1]
```diff
@@ -1 +1,2 @@
-LTO_ENABLE = yes
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/keymaps/default/rules.mk:1]: This file should be removed.

File: keyboards/clickety_split/leeloo/keymaps/default_led/rules.mk [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Intentionally left blank.
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/keymaps/default_led/rules.mk:1]: This file should be removed.

File: keyboards/clickety_split/leeloo/info.json [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
-    "device_version": "0.1.3"
+    "pid": "0x2022"
   },
+  "diode_direction": "COL2ROW",
+  "matrix_pins": {
+      "cols": ["F6", "F7", "B1", "B3", "B2", "B6"],
+      "rows": ["C6", "D7", "E6", "B4", "B5"]
+  },
+  "split": { "soft_serial_pin": "D2" },
+  "tapping": { "term": 150 },
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/info.json:15]: Generally not preferred to set this at the keyboard level, especially lowering it from the default.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/clickety_split/leeloo/rules.mk [hunk 4]
```diff
@@ -6,16 +6,31 @@ BOOTLOADER = caterina
...
+# Enables the use of OLED displays
+OLED_ENABLE = yes
+OLED_DRIVER = SSD1306
+
+# Enable LEDs
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+RGB_MATRIX_ENABLE = no      # Enable keyboard RGB matrix (do not use together with RGBLIGHT_ENABLE)
+RGB_MATRIX_DRIVER = WS2812  # RGB matrix driver support
+RGB_MATRIX_SUPPORTED = no
+
```
[the remark this answers is not in the document]
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/rules.mk]: Thank you @waffle87!

File: keyboards/clickety_split/leeloo/rev2/config.h [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+#define MATRIX_COLS 6
+
+/* encoder support */
+#define ENCODERS_PAD_A { F5 }
+#define ENCODERS_PAD_B { F4 }
+#define ENCODERS_PAD_A_RIGHT { F4 }
+#define ENCODERS_PAD_B_RIGHT { F5 }
+
+#define RGB_DI_PIN       D3
+#define DRIVER_LED_TOTAL 74
```
[the remark this answers is not in the document]
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/rev2/config.h]: Thank you @waffle87!

File: keyboards/clickety_split/leeloo/keymaps/default/rules.mk (hunk 1 above)
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/keymaps/default/rules.mk:1]: Thank you @waffle87, I'll remove the file.

File: keyboards/clickety_split/leeloo/keymaps/default_led/rules.mk (hunk 2 above)
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/keymaps/default_led/rules.mk:1]: Thank you @waffle87, I'll remove this file as well.

File: keyboards/clickety_split/leeloo/info.json (hunk 3 above)
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/info.json:15]: Thank you, @waffle87; I'll remove the tapping term.

File: keyboards/clickety_split/leeloo/readme.md [hunk 6]
```diff
@@ -1,16 +1,33 @@
...
+
+Leeloo v2 is the evolution of Leeloo, which is a 4x6x5m ortholinear split keyboard designed, created, and sold by Clickety Split Ltd.
+
+* Keyboard Maintainer: [Clickety Split](https://github.com/ClicketySplit)
+* Hardware Supported: Leeloo PCB rev1/rev2, Pro Micro, Elite-C
+* Hardware Availability: [clicketysplit.ca/pages/leeloo](https://clicketysplit.ca/pages/leeloo)
+
 # Leeloo
 
 ![Leeloo](https://cdn.shopify.com/s/files/1/0599/3460/5491/files/Leeloo-rev1.0-w.jpg?v=1646798726)
```
Comment (drashna) [inline keyboards/clickety_split/leeloo/readme.md:13]: please rehost the image.

File: keyboards/clickety_split/leeloo/leeloo.c [hunk 7]
```diff
@@ -46,19 +46,32 @@ static void render_mod_status(uint8_t modifiers) {
...
+#if defined(KEYBOARD_clickety_split_leeloo_rev2)
+static void render_secondary_oled(void) {
+    // Version Information
+    oled_write_P(PSTR("Leeloo v2\n\n"), false);
+    oled_write_P(PSTR("Firmware: "), false);
+    oled_write_P(PSTR("v2.13"), false);
+    oled_write_P(PSTR("\n"), false);
+    oled_write_P(PSTR("Clickety Split Ltd."), false);
+}
+#endif
```
[the remark this answers is not in the document]
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/leeloo.c:69]: Thank you very much @drashna!

File: keyboards/clickety_split/leeloo/readme.md [hunk 8]
```diff
@@ -1,16 +1,33 @@
+# Clickety Split | Leeloo v2
+
+![Leeloo v2](https://cdn.shopify.com/s/files/1/0599/3460/5491/files/Leeloo-v2-Wired.jpg?v=1669873125)
```
[the remark this answers is not in the document]
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/readme.md]: Hi @drashna!

I hope it's okay that I hosted the images on Clickety Split's GitHub.

:)

File: keyboards/clickety_split/leeloo/readme.md (hunk 6 above)
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/readme.md]: Likewise for this one.

File: keyboards/clickety_split/leeloo/readme.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (waffle87) [reply keyboards/clickety_split/leeloo/readme.md]: Needs to Imgur — Github links change, repositories are deleted, etc. and the image will be broken.

[the remark this answers is not in the document]
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/readme.md]: Thank you @waffle87!  I've posted to Imgur; and, I'll update the readme.md file.

**Review by drashna — APPROVED** [review]

File: keyboards/clickety_split/leeloo/rules.mk [hunk 9]
```diff
@@ -6,16 +6,25 @@ BOOTLOADER = caterina
 
 # Build Options
 #   change yes to no to disable
-#
+EXTRAKEY_ENABLE = yes       # Audio control and System control
 BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
 MOUSEKEY_ENABLE = no        # Mouse keys
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/rules.mk:11]:
```suggestion
#
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
```

File: keyboards/clickety_split/leeloo/rules.mk [hunk 10]
```diff
@@ -6,16 +6,25 @@ BOOTLOADER = caterina
...
-EXTRAKEY_ENABLE = no        # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
 NKRO_ENABLE = no            # Enable N-Key rollover
-BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 AUDIO_ENABLE = no           # Audio output
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/rules.mk:17]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/clickety_split/leeloo/rules.mk (hunk 10 above)
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/rules.mk]: Thank you @fauxpark.

**Comment by ClicketySplit** [comment]
I'm sorry about the confusion @waffle87 and @fauxpark!  I mistakenly pressed the wrong button.

File: keyboards/clickety_split/leeloo/rev2/config.h [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+#pragma once
+
+#include "config_common.h"
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 6
+
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/rev2/config.h:26]:
```suggestion
```

File: keyboards/clickety_split/leeloo/readme.md [hunk 12]
```diff
@@ -1,16 +1,33 @@
...
+
+The original design and release of Leeloo.
+
+# Sample Build Commands for Leeloo v1 and v2
 
-Make example, after setting up build environment:
+After setting up the build environment, consider the following Make commands:
 
+Leeloo v1:
+```
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/readme.md:22]:
```suggestion
```

File: keyboards/clickety_split/leeloo/readme.md [hunk 13]
```diff
@@ -1,16 +1,33 @@
...
     make clickety_split/leeloo/rev1:default
+    make clickety_split/leeloo/rev1:default:flash
+```
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/readme.md:25]:
```suggestion
    make clickety_split/leeloo/rev2:default
```

File: keyboards/clickety_split/leeloo/readme.md [hunk 14]
```diff
@@ -1,16 +1,33 @@
...
 
+Leeloo v2:
+```
+    make clickety_split/leeloo/rev2:default_led
+    make clickety_split/leeloo/rev2:default_led:dfu
+```
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/readme.md:31]:
```suggestion
```

File: keyboards/clickety_split/leeloo/readme.md [hunk 15]
```diff
@@ -1,16 +1,33 @@
...
 
-See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with the [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/readme.md:33]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/clickety_split/leeloo/readme.md [hunk 16]
```diff
@@ -1,16 +1,33 @@
...
+# Leeloo
+
+![Leeloo](https://i.imgur.com/x1ew17D.jpg)
+
+The original design and release of Leeloo.
+
+# Sample Build Commands for Leeloo v1 and v2
 
-Make example, after setting up build environment:
+After setting up the build environment, consider the following Make commands:
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/readme.md:19]:
```suggestion
Make example for this keyboard (after setting up your build environment):
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by ClicketySplit** [comment]
Thank you very much for the recommendations, @fauxpark!

File: keyboards/clickety_split/leeloo/rev1/config.h [hunk 17]
```diff
@@ -24,38 +24,8 @@
 #define MATRIX_ROWS 10
 #define MATRIX_COLS 6
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/rev1/config.h:25]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/clickety_split/leeloo/rev1/config.h [hunk 18]
```diff
@@ -21,41 +21,9 @@
 
 /* key matrix size */
 // Rows are doubled-up
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/rev1/config.h:23]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/clickety_split/leeloo/info.json [hunk 19]
```diff
@@ -1,13 +1,18 @@
...
-    "pid": "0x2022",
-    "device_version": "0.1.3"
+    "pid": "0x2022"
   },
+  "diode_direction": "COL2ROW",
+  "matrix_pins": {
+      "cols": ["F6", "F7", "B1", "B3", "B2", "B6"],
+      "rows": ["C6", "D7", "E6", "B4", "B5"]
+  },
+  "split": { "soft_serial_pin": "D2" },
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/info.json:14]:
```suggestion
  "encoder": {
    "rotary": [
      {"pin_a": "F5", "pin_b": "F4"}
    ]
  },
  "split": {
      "soft_serial_pin": "D2",
      "encoder": {
          "right": {
              "rotary": [
                  {"pin_a": "F4", "pin_b": "F5"}
              ]
          }
      }
  },
```

File: keyboards/clickety_split/leeloo/rev1/config.h [hunk 20]
```diff
@@ -19,43 +19,8 @@
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/rev1/config.h:19]: This file can be deleted with the above suggestion.

File: keyboards/clickety_split/leeloo/rev2/config.h [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* encoder support */
+#define ENCODERS_PAD_A { F5 }
+#define ENCODERS_PAD_B { F4 }
+#define ENCODERS_PAD_A_RIGHT { F4 }
+#define ENCODERS_PAD_B_RIGHT { F5 }
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/rev2/config.h:26]:
```suggestion
```

File: keyboards/clickety_split/leeloo/rev2/config.h [hunk 22]
```diff
@@ -0,0 +1,44 @@
...
+
+#define RGB_DI_PIN       D3
+#define RGB_MATRIX_LED_COUNT 74
+
+#ifdef RGBLIGHT_ENABLE
+#    define RGBLED_SPLIT \
+        { 37, 37 }
+#    define RGBLIGHT_SPLIT
+#endif
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/rev2/config.h:35]:
```suggestion
```

File: keyboards/clickety_split/leeloo/rev2/rules.mk [hunk 23]
```diff
@@ -0,0 +1,5 @@
+# Enable LEDs
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+RGB_MATRIX_ENABLE = yes     # Enable keyboard RGB matrix (do not use together with RGBLIGHT_ENABLE)
+RGB_MATRIX_DRIVER = WS2812  # RGB matrix driver support
+RGB_MATRIX_SUPPORTED = yes
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/rev2/rules.mk:5]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by ClicketySplit** [comment]
Hi @waffle87,

I was able to go through the process of committing the recommendations you have made with successful build—on my local machine and what QMK CI Build shares back.  However, I see that there are a couple of conflicts that I cannot seem to resolve.  May I ask you to have a closer look, please?  I think it has to do with the merge into master.

Thank you!

File: keyboards/clickety_split/leeloo/info.json [hunk 24]
```diff
@@ -1,13 +1,32 @@
...
+      "soft_serial_pin": "D2",
+      "encoder": {
+          "right": {
+              "rotary": [
+                  {"pin_a": "F4", "pin_b": "F5"}
+              ]
+          }
+      }
+  },
+  "debounce": 5,
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/info.json:29]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/clickety_split/leeloo/info.json (hunk 24 above)
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/info.json]: Thank you for the recommendation, @fauxpark!

**Comment by ClicketySplit** [comment]
Hi @fauxpark!

Thank you for your recommendation, it seems like there is only one file that needs to be resolved.  It looks like the compile is successful and when I try to compile on my local, it is also successful.  Would you have any insight to what may allow the conflict to be resolved?

Thank you for your help!

**Comment by fauxpark** [comment]
I've fixed the conflict. Technically the `TAPPING_TERM` can go into info.json as well, then you can delete config.h completely.

File: keyboards/clickety_split/leeloo/info.json [hunk 25]
```diff
@@ -1,12 +1,30 @@
...
+  },
+  "split": {
+      "soft_serial_pin": "D2",
+      "encoder": {
+          "right": {
+              "rotary": [
+                  {"pin_a": "F4", "pin_b": "F5"}
+              ]
+          }
+      }
```
Comment (fauxpark) [inline keyboards/clickety_split/leeloo/info.json:27]:
```suggestion
```
Duplicate

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by ClicketySplit** [comment]
Thank you for helping me fix the info.json and TAPPING_TERM, @fauxpark!

:)

File: keyboards/clickety_split/leeloo/keymaps/default_led/config.h [hunk 26]
```diff
@@ -0,0 +1,58 @@
...
+ * 
+ * For a complete list, visit: https://docs.qmk.fm/#/feature_rgb_matrix?id=rgb-matrix-effects.
+ * Some of the effects take up a lot of memory, so you may need to manage the number of
+ * effects that are loaded at any given time.
+*/
+    #define ENABLE_RGB_MATRIX_GRADIENT_UP_DOWN
+    #define ENABLE_RGB_MATRIX_SOLID_REACTIVE_NEXUS
+    #define ENABLE_RGB_MATRIX_SPLASH
+    
+#endif
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/keymaps/default_led/config.h:58]: Ideally this is just moved to `keymaps/default/config.h`, and this directory removed.

File: keyboards/clickety_split/leeloo/rev2/config.h [hunk 27]
```diff
@@ -0,0 +1,30 @@
...
+#define RGB_MATRIX_LED_COUNT 74
+
+/* RGB matrix support */
+#ifdef RGB_MATRIX_ENABLE
+#    define SPLIT_TRANSPORT_MIRROR
+#    define RGB_MATRIX_SPLIT \
+        { 37, 37 }
+#    define RGB_MATRIX_MAXIMUM_BRIGHTNESS 115
+#    define RGB_DISABLE_WHEN_USB_SUSPENDED
+#endif
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/rev2/config.h:30]:
```suggestion
#ifdef RGB_MATRIX_ENABLE
#    define RGB_MATRIX_LED_COUNT 74
#    define RGB_DISABLE_WHEN_USB_SUSPENDED
#endif
```

File: keyboards/clickety_split/leeloo/rev2/info.json [hunk 28]
```diff
@@ -0,0 +1,6 @@
+{
+  "keyboard_name": "Leeloo v2",
+  "usb": {
+    "device_version": "0.2.0"
+  }
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/rev2/info.json:5]:
```suggestion
  },
  "features": {
      "rgb_matrix": true
  },
  "split": {
      "transport": {
          "sync_matrix_state": true
      }
  },
  "ws2812": {
      "pin": "D3"
  },
  "rgb_matrix": {
      "driver": "WS2812",
      "max_brightness": 115,
      "split_count": [37, 37]
  }
```

File: keyboards/clickety_split/leeloo/rev2/rev2.c [hunk 29]
```diff
@@ -0,0 +1,117 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "rev2.h"
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/rev2/rev2.c:18]:
```suggestion
#include "quantum.h"
```

File: keyboards/clickety_split/leeloo/rev2/rev2.h [hunk 30]
```diff
@@ -0,0 +1,40 @@
...
+		{ L10, L11, L12, L13, L14, L15 }, \
+		{ L20, L21, L22, L23, L24, L25 }, \
+		{ L30, L31, L32, L33, L34, L35 }, \
+		{ KC_NO, L41, L42, L43, L44, L45 }, \
+		{ R00, R01, R02, R03, R04, R05 }, \
+		{ R10, R11, R12, R13, R14, R15 }, \
+		{ R20, R21, R22, R23, R24, R25 }, \
+		{ R30, R31, R32, R33, R34, R35 }, \
+		{ KC_NO, R41, R42, R43, R44, R45 } \
+	}
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/rev2/rev2.h:40]: This should be migrated to `info.json`.

File: keyboards/clickety_split/leeloo/rev2/rules.mk [hunk 31]
```diff
@@ -0,0 +1,4 @@
+# Enable LEDs
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+RGB_MATRIX_ENABLE = yes     # Enable keyboard RGB matrix (do not use together with RGBLIGHT_ENABLE)
+RGB_MATRIX_DRIVER = WS2812  # RGB matrix driver support
```
Comment (waffle87) [inline keyboards/clickety_split/leeloo/rev2/rules.mk:4]:
```suggestion
# This file intentionally left blank
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by waffle87** [comment]
Keyboard updates are also required to target the `develop` branch. Looks like the only conflict that needs to be resolved is in ` keyboards/clickety_split/leeloo/leeloo.h `.

File: keyboards/clickety_split/leeloo/keymaps/default_led/config.h (hunk 26 above)
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/keymaps/default_led/config.h:58]: Thank you for your feedback, @waffle87!  I will migrate this over to default and remove this directory.

File: keyboards/clickety_split/leeloo/rev2/rev2.h (hunk 30 above)
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/rev2/rev2.h:40]: Thank you @waffle87!  I will migrate the snippet over to info.json.

Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/rev2/rev2.h]: Hi @waffle87,

I was able to migrate the Layout to info.json; thank you for the recommendation, it certainly makes things cleaner.

File: keyboards/clickety_split/leeloo/keymaps/default_led/config.h (hunk 26 above)
Comment (ClicketySplit) [reply keyboards/clickety_split/leeloo/keymaps/default_led/config.h]: Migrated and files have been removed, @waffle87. :)

**Pull request closed without merging**